### DISCUSSION TITLE
Mobile Touch Unit

### DISCUSSION THREAD

**Discussion by Telemachus1977** [tagyoureit/nodejs-poolController#939] [General]
My next door neighbor just scrapped his mobiletouch unit. 

PH4Mohiletouch2A, along with the entire front control panel of his intellitouch.

I think you he have upgraded to an intelli Center, but I'm not sure.  I grabbed all the leftover parts that his pool guy left. 

Mobile touch unit, antenna unit, and the entire front panel and control board from the intellitouch.

At any rate, is there any way that this mobile touch 2 could be integrated with a pixie controller?

I also have the entire circuit board from the intellitouch...

**Comment by tagyoureit** [comment]
The mobile touch acts like a second OCP, and you'd likely need to have it be the master on the system because of it's logic.  It's been around a while.  You would be better off just sticking with Nixie, it's way more functional.

**Reply by rstrouse** [reply]
Actually the mobile touch is an indoor control panel with a radio.  It will not operate without an IntelliTouch outdoor control panel and transceiver.

**Reply by Telemachus1977** [reply]
Talking with my neighbor, he replaced it all because the antenna stopped working for the mobile touch. 

I think I will hang on to the intellitouch board. I might end up using it if my easy touch ever dies. 

Maybe the antenna for my easy touch can be modified to work with the mobile touch...
